bin/docserv-stitch

```diff
@@ -131,13 +140,33 @@ done
 ([[ "$simplify_config" -eq 1 ]] && [[ -z "$output_file" ]]) && \
   out "--simplify can only be used together with an OUTPUT_FILE parameter."
 
+([[ "$revalidate" -eq 1 ]] && [[ -z "$output_file" ]]) && \
+  out "--revalidate-only can only be used together with an OUTPUT_FILE parameter."
+
 cd $input_dir
 
+[[ ! $(ls *.xml 2>/dev/null) ]] && \
+  out "There are no product configuration files."
+
+
 valid_languages_sorted=''
 [[ "$valid_languages" ]] && valid_languages_sorted=$(echo -e "$valid_languages" | tr ' ' '\n' | sort -u)
 
 issuelist=''
 
+
+# Create md5 hashes for all config files. From the md5sum output, we are only
+# keeping the hashes, as the file names of the config files are actually
+# irrelevant.
+hashsums=$(md5sum *.xml | sort | sed -r 's/\s*[^ ]+$//' | tr '\n' '!')
+
+if [[ "$revalidate" == '1' ]]; then
+  if [[ -f $output_file ]]; then
```

Comment:
Couldn't/shoudn't the two if statements combined using an and?

Reply:
They could. Thanks, Toms! Fixed as well. (Not on master yet but soon will be.)